source/math/FloatTransform3D.ooc

```diff
@@ -208,7 +208,7 @@ FloatTransform3D: cover {
 	}
 	transformAndProject: func ~FloatPoint2D (point: FloatPoint2D, focalLength: Float) -> FloatPoint2D {
 		transformedWorldPoint := this * FloatPoint3D new(point x, point y, focalLength)
-		this project(transformedWorldPoint, focalLength)
+		focalLength == 0.0f ? FloatPoint2D new(transformedWorldPoint x, transformedWorldPoint y) : this project(transformedWorldPoint, focalLength)
```

Comment:
I mean kean does not have NaN check. 
You are right, if we know somewhere may cause NaN, this simple check can be placed there.
It's not a problem now. But maybe it's better to have one "isNumber()" in the future as Simon suggested.